### ISSUE TITLE
When user is in Easel in Creator Hub viewing 3D NFT viewing Full Image- the  3 doted menu option does not appear

### ISSUE THREAD

**Issue by Tanyaml** [Pylons-tech/pylons#824]
Actual Results: 
When user is Easel and creates a Draft & Published when viewing the Full Image the 3 doted menu options does not appear to click for Publish, Delete and Copy CID

Expected Results:
When user is in in Easel in Creator Hub in the Draft tab and  viewing the Full Image the 3 doted menu options SHOULD appear to click for Publish, Delete and Copy CID when user is in Draft and when user is in Creator hub in the Publish Tab you can view 2 doted menu for Publish and Delete.

Please see video:

https://user-images.githubusercontent.com/93543576/185992996-a1e93a77-09a8-4315-a40e-ffa73a213efc.mov

## Steps to Reproduce
1. User creates 3D NFT
2. User  is on Creator Hub and clicks on draft and views Full image of NFT and issue where the 3 dotes menu option does not appear where the Publish, Delete and Copy CID is an option
3. User continues to publish 3D NFT and is published
4. User is in Creator Hub and clicks on Publish and views Full Image of NFT and issue where 2 doted menu options does not appear where the Publish, Delete is an option

____

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by StefanB90** [comment]
@faizhashmiRnS this is because of the white background associated with that specific 3D NFT referenced above. @christine addressed this issue when she made the background icons for the banner image in the Pylons app change to a darker color based on the background color. Please leverage this code she created and implement it in Easel to address this issue.

**Comment by MikeSofaer** [comment]
I think the whole image should be the click target for the menu.

**Comment by faizhashmiRnS** [comment]
@StefanB90 Agreed. But if we set dark color in case of 3D, this will work fine with 3D. But if what if we have a white thumbnail for Audio, Video or a White Image NFT. The white three dot menu color will still be invisible.

**Comment by MikeSofaer** [comment]
This is why the whole image should be the click target

**Comment by StefanB90** [comment]
Resolved: for all thumbnails in Draft section --> remove three dots and make the whole image clickable

**Comment by faizhashmiRnS** [comment]
@StefanB90 should it be same in case of Published NFTs section?

**Comment by faizhashmiRnS** [comment]
@kjawadDeveloper2 I have made the whole grid clickable and further in case of 3Ds I have set the background color a bit grey.

Does this need any changes and should it be same for Published NFTs and Drafts?

**Comment by kjawadDeveloper2** [comment]
Revert it back to white background. And make the items of the Grid layout clickable.

**Comment by StefanB90** [comment]
when you set your price.. should take you directly to published preview moment. Shouldn't have to click three dots menu to then be able to click another option to finally get to where you can delist & resist your NFT.

@StefanB90 to update issue for clarity and assign after requirements are clear.

**Comment by StefanB90** [comment]
@Tanyaml let's test this to see if we can close it. Thank you.

**Comment by Tanyaml** [comment]
I have validated this although do we still need the three dots being that you can just click on the NFT @StefanB90

**Comment by StefanB90** [comment]
@faizhashmiRnS please remove the three dots in the Creator Hub cards since the entire card is now clickable. Once you are able to do that, we can close this issue. Thank you for your help.

CC @Tanyaml

**Comment by faizhashmiRnS** [comment]
> @faizhashmiRnS please remove the three dots in the Creator Hub cards since the entire card is now clickable. Once you are able to do that, we can close this issue. Thank you for your help.
> 
> CC @Tanyaml

Yes, Just did this in PR #1072
